lib/macho/utils.rb

```diff
@@ -33,4 +33,16 @@ def self.magic32?(num)
   def self.magic64?(num)
     num == MH_MAGIC_64 || num == MH_CIGAM_64
   end
+
+  # @param num [Fixnum] the number being checked
+  # @return [Boolean] true if `num` is a valid little-endian magic number, false otherwise
+  def self.little_magic?(num)
+    num == FAT_CIGAM || num == MH_CIGAM || num == MH_CIGAM_64
+  end
+
+  # @param num [Fixnum] the number being checked
+  # @return [Boolean] true if `num` is a valid big-endian magic number, false otherwise
+  def self.big_magic?(num)
+    num == FAT_MAGIC || num == MH_CIGAM || num == MH_CIGAM_64
```

Comment:
You can drop `FAT_MAGIC` here. Basically the same reasons like in `MachO.little_magic?`.